Missions:
 Mission_1:
   going_road: 2
   head_angle: 0
  
 Mission_2:
   going_road: 3
   head_angle: 90
  
 Mission_3:
  going_road: 1
  head_angle: 180
  
 Mission_4:
  going_road: 5
  head_angle: 90
